CHANGELOG.md

```diff
@@ -1,3 +1,80 @@
+3.27.0
+    - Added evaluation order option in body agent control (ENT-13295)
+	- Added findlocalgroups policy function (CFE-4550)
+	- Added getgroupinfo policy function (CFE-4512)
+	- Added getgroups policy function (ENT-12722)
+	- Added isnewerthantime policy function (CFE-2815)
+	- Added missing record changes/failures to when flipping the immutable bit
+	  (ENT-13179)
+	- Added policy function classfilterdata() (CFE-3421, ENT-6193)
+	- Added policy function to get ACLs (CFE-4529)
+	- Added sys policy version variable (ENT-4043)
+	- Added sysvinit cf-php-fpm service script for Mission Portal (ENT-13234)
+	- Atomic permissions during file copy
+	  Temporary file is now set to promised permissions before replacing it
+	  with original during remote copy from (ENT-13163)
+	- Files promise can now modify immutable bit in file system attributes
+	  (CFE-1840, ENT-10961)
+	- Fixed assertion error in classfilterdata()
+	  An assertion was triggered if the variable name or inline JSON failed to
+	  expand when calling classfilterdata().
+	- Fixed bug in getacls() where no ACLs caused error
```

Comment:
This bugfix changelog is not needed right? Since we are fixing a bug in something introduced in this version? @larsewi ?